manifests/a/AVerMedia/AssistCentral/1.2.0.13/AVerMedia.AssistCentral.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AVerMedia.AssistCentral
+PackageVersion: 1.2.0.13
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft

```

Comment:
Installer does not install silently in a sandbox environment; Silent switches must be provided for the package